install.sh

```diff
@@ -11,6 +11,8 @@ fi
 UBUNTU_PRE_2004=false
 if $UBUNTU; then
 	LSB_RELEASE=$(lsb_release -rs)
+	# In case Ubuntu minimal does not come with bc
+	if [ $(which bc |wc -l) -eq 0 ]; then apt install bc -y; fi
```

Comment:
Needs quotes around the $which... `	if [ "$(which bc |wc -l)" -eq 0 ]; then apt install bc -y; fi`